src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -841,4 +841,31 @@ public int hashCode() {
 
 	}
 
+	@Nullable
+	// Converts string to a valid UUID string (32 alphanumeric and 4 dashes, 36)
+	// Example: Inserting "1" converts to "00000000-00000000-00000000-00000001"
```

Comment:
rather than simple comments let's use some javadocs to make this clearer and move it above the `@Nullable` annotation or move that next to string `public static @Nullable String`